src/arm/96boards.c

```diff
@@ -304,7 +304,7 @@ mraa_96boards()
             ls_gpio_pins = rock960_ls_gpio_pins;
             b->uart_dev[0].device_path = (char*) rock960_serialdev[0];
             b->uart_dev[1].device_path = (char*) rock960_serialdev[1];
-        } else if (mraa_file_contains(DT_BASE "/model", "ZynqMP ZCU100 RevC")) {
+        } else if ((mraa_file_contains(DT_BASE "/model", "ZynqMP ZCU100 RevC"))||(mraa_file_contains(DT_BASE "/model", "Avnet Ultra96 Rev1"))) {
```

Comment:
This line is too long for mraa's code style - to make it simple, please run clang-format on the updated file with the configuration file from mraa repo root.

Reply:
Resolved in current patch version, thanks!